webserver/views/user.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 import webserver
 import db.user
+import ujson
```

Comment:
`ujson` gave us problems after about 10 decimal places on floating point numbers. We started using it in ListenBrainz because we were serialising and deserialising a lot of data and it's about 3x faster than regular `json`. For this task it's OK to continue using `ujson`